website/tfe.erb

```diff
@@ -36,6 +36,9 @@
                         <li<%= sidebar_current("docs-datasource-tfe-workspace-ids") %>>
                             <a href="/docs/providers/tfe/d/workspace_ids.html">tfe_workspace_ids</a>
                         </li>
+                        <li<%= sidebar_current("docs-datasource-tfe-oauth-client") %>>
+                            <a href="/docs/providers/tfe/d/oauth_client.html">tfe_oauth_client</a>
+                        </li>
```

Comment:
nit: this should be moved up above organization membership so things are still in alphabetical order